otter/test/convergence/test_steps.py

```diff
@@ -469,9 +469,9 @@ def test_add_nodes_to_clb_terminal_failures(self):
         if there is any other 4xx error, then
         the error is propagated up and the result is a failure.
         """
-        terminals = (CLBDeletedError(lb_id=u"12345"),
+        terminals = (CLBNotFoundError(lb_id=u"12345"),
+                     CLBDeletedError(lb_id=u"12345"),
                      CLBNodeLimitError(lb_id=u"12345"),
-                     NoSuchCLBError(lb_id=u"12345"),
```

Comment:
Oh, ok, I was confused because you suggested leaving `CLBDeletedError`, and that's exactly the error which I _did_ leave in the test :-) So I guess you mean I should add `NoSuchCLBError` back, to cover both subclasses and the superclass?